middle_end/flambda/inlining/inlining_decision.ml

```diff
@@ -126,6 +134,10 @@ module Function_declaration_decision = struct
         Code_size.print small_function_size
         Code_size.print size
         Code_size.print large_function_size
+    | Functor { size } ->
+      Format.fprintf fmt "this@ function@ is@ a@ functor@ and@ might@ be@ \
+                          inlined@ independently@ from@ it's@ size (%a)."
```

Comment:
Maybe: "the function is a functor (so the large function threshold was not applied)"